extension/src/server/interactionService.ts

```diff
@@ -91,57 +92,58 @@ export class InteractionService implements IInteractionService {
         if (result === no) {
             return false;
         }
+
         return null;
     }
 
     async promptForSelection(promptText: string, choices: string[]): Promise<string | null> {
         const selected = await vscode.window.showQuickPick(choices, {
-            placeHolder: promptText,
+            placeHolder: formatText(promptText),
             canPickMany: false,
             ignoreFocusOut: true
         });
 
-        return selected || null;
+        return selected ?? null;
     }
 
-    displayIncompatibleVersionError(requiredCapability: string, appHostHostingSdkVersion: string) {
-        const cliInformationalVersion = 'Unknown'; // Replace with actual CLI version if available
+    async displayIncompatibleVersionError(requiredCapabilityStr: string, appHostHostingSdkVersion: string, rpcClient: ICliRpcClient) {
+        const cliInformationalVersion =  await rpcClient.getCliVersion();
 
         const errorLines = [
-            "The app host is not compatible. Consider upgrading the app host or Aspire CLI.",
-            "",
-            `\tAspire Hosting SDK Version: ${appHostHostingSdkVersion}`,
-            `\tAspire CLI Version: ${cliInformationalVersion}`,
-            `\tRequired Capability: ${requiredCapability}`,
-            ""
+            incompatibleAppHostError,
+            '',
+            `\t${aspireHostingSdkVersion(appHostHostingSdkVersion)}`,
+            `\t${aspireCliVersion(cliInformationalVersion)}`,
+            `\t${requiredCapability(requiredCapabilityStr)}`,
+            ''
         ];
 
         errorLines.forEach(line => {
-            vscode.window.showErrorMessage(line);
-            this._outputChannelWriter.appendLine(line);
+            vscode.window.showErrorMessage(formatText(line));
```

Comment:
What does this visually look like? Are you popping an error message for each line?

Reply:
Currently looks like this: (newlines cannot be rendered in showErrorMessage)
<img width="476" alt="image" src="https://github.com/user-attachments/assets/aef62445-0f22-47a6-97a6-a8a984524d2f" />

I've changed it to this now
<img width="476" alt="image" src="https://github.com/user-attachments/assets/a9d97875-b125-4ead-87d7-260e780da725" />